workshops/create-custom-supply-chains.yaml

```diff
@@ -0,0 +1,294 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:base64", "base64")
+#@ load("@ytt:json", "json")
+---
+apiVersion: learningcenter.tanzu.vmware.com/v1beta1
+kind: Workshop
+metadata:
+  name: scc-workshop-dev
+spec:
+  title: Supply Chain Choreographer - Development Version
+  description: A workshop that demonstrates the capabilities Supply Chain Choreographer provides and how to customize the supply chains.
+  vendor: tanzu.vmware.com
+  authors:
+    - Timo Salm
+    - Gary Gutman
+    - Deepak Khopade
+  #! The difficulty field indicates the target audiences of the workshop. 
+  #! The value can be: beginner, intermediate, advanced, or extreme.
+  difficulty: intermediate
+  #! The duration field gives the maximum amount of time the workshop takes to complete. 
+  #! This field provides informational value and does not guarantee how long a workshop instance lasts. 
+  #! The field format is an integer number with s, m, or h suffix.
+  duration: 3h
+  #! url: https://github.com/mrgaryg/tap-cartographer-workshop
+  #! url: #@ data.values.workshop_git.git_url + "/" + data.values.workshop_git.git_org_name + "/" + data.values.workshop_git.git_repo
+  url: https://github.com/dkhopade/tap-cartographer-workshop
+  content:
+    image: #@ "{}/tap-workshop/cartographer-workshop:latest".format(data.values.container_registry.hostname)
+    files: #@ "{}/{}/tap-cartographer-workshop/workshop?ref=se-supplychains-workshop-dev".format(data.values.workshop_git.git_url,data.values.workshop_git.git_org_name)
+  session:
+    resources:
+      memory: 3Gi
+    namespaces:
+      budget: custom
+    applications:
+      terminal:
+        enabled: true
+        layout: split
+      console:
+        enabled: true
+        vendor: octant
+      editor:
+        enabled: true
+    dashboards:
+      - name: Cartographer Docs
+        url: https://cartographer.sh/docs/development/
+      - name: TAP GUI
+        url: #@ "http://tap-gui.{}/supply-chain/".format(data.values.tap.ingress_domain)
+      - name: Carto Live Editor
+        url: https://cartographer.sh/live-editor/index.htm
+    env: 
+    - name: TAP_INGRESS
+      value: #@ data.values.tap.ingress_domain
+    - name: TAP_CNRS_SUBDOMAIN
+      value: #@ data.values.tap.cnrs_subdomain
+    - name: CONTAINER_REGISTRY_HOSTNAME
+      value: #@ data.values.container_registry.hostname
+    - name: CONTAINER_REGISTRY_USERNAME
+      value: #@ data.values.container_registry.username
+    - name: CONTAINER_REGISTRY_PASSWORD
+      value: #@ data.values.container_registry.password
+    - name: CONTAINER_REGISTRY_REPOSITORY
+      value: #@ data.values.container_registry.repository
+    - name: GITOPS_REPOSITORY
+      value: #@ "{}/$(session_namespace).git".format(data.values.gitops.https_repository_prefix)
+    - name: GITOPS_REPOSITORY_USERNAME
```

Comment:
The user name for gitea is "gitea_admin" and password for "gitea_admin" can be injected by referencing `data.values.gitea.adminPassword`